src/generic/Loading.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Spinner } from '@edx/paragon';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+
+export default function Loading() {
+  return (
+    <div
+      className="d-flex justify-content-center align-items-center flex-column"
+      style={{
+        height: '50vh',
+      }}
+      data-test-id="spinnerContainer"
+    >
+      <Spinner className animation="border" role="status" variant="primary">
+        <span className="sr-only">
+          <FormattedMessage
+            id="authoring.loading"
+            defaultMessage="Loading..."
+            description="Screen-reader message for when a page is loading."
```

Comment:
Just wondering if we have anything generic in the frontend build? 
also, can we move these enums to the messages.js?

Don't these need internationalization?

Reply:
<FormattedMessage> is a react-intl class that does internationalization - it's an alternative to putting messages in the messages.js file.  The job which uploads messages to transifex grabs all the messages.js files _and_ scans for any react-intl components (like FormattedMessage) and pulls them in too.  

So this is in a good place and is internationalized. :)

As a side note, frontend-build is just build tools - I think you might mean frontend-platform.  There's a `PageLoading` component in there, but the i18n aspects of it are kind of problematic since it's, ironically enough to this conversation, not internationalized properly.

Reply:
Right so because we have used react-intl class these messages won't be picked up by transifex? correct? but that's fine as long as we have internationalization. 

- No plans on updating the `PageLoading` component in the frontend-platform?  I did mean platform, thank you :) 

just questions, no blockers 👍